plugins/rbac/src/components/CreateRole/AddMembersForm.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import { useAsync } from 'react-use';
+
+import { useApi } from '@backstage/core-plugin-api';
+
+import { LinearProgress, TextField } from '@material-ui/core';
```

Comment:
I mentioned this on @debsmita1's PR but probably good to mention it here too.  I think we should try and use Material 5 whenever possible, so `@mui/material/LinearProgress` , `@mui/material/TextField` and so on.  It could be too late to do this for this particular iteration and that's fine, we'll have to do it eventually when upstream drops Material 4.